test/irb/test_context.rb

```diff
@@ -50,22 +50,20 @@ def test_evaluate_with_exception
     end
 
     def test_evaluate_with_encoding_error_without_lineno
-      pend if RUBY_ENGINE == 'truffleruby'
-      assert_raise_with_message(EncodingError, /invalid symbol/) {
-        @context.evaluate(%q[{"\xAE": 1}], 1)
+      # TruffleRuby adds 'encoding' in the middle of the message
```

Comment:
This will be fixed with https://github.com/oracle/truffleruby/issues/2850. I expect that'll change fairly quickly, so you may not want to modify the test. Of course, we can always change the test back once the message is changed in TruffleRuby.

Reply:
👍 since the issue has been addressed, I left the assertion unchanged.